vuepress/docs/v7.0/tutorials/devops/external-id-management.md

```diff
@@ -2,75 +2,83 @@
 
 ## Purpose
 
-This tutorial enables the Entando administrator to leverage an existing
-Keycloak instance.
+This tutorial details how an Entando instance can be connected to an existing Keycloak instance by an admin user of the Keycloak instance.
 
-## Requirements
+## Prerequisites
 
 -   A Keycloak instance
-
--   A realm named "entando" on that instance
-
--   An admin user for the "entando" realm
+-   A realm named "entando" in that instance
+-   Admin user credentials for the "entando" realm
 
 ## Steps
 
-### 1. Get keycloak information
-
-Retrieve the relevant information from the Keycloak instance you want to
-use.
-
-Specifically you will need:
+### 1. Get Keycloak information
 
--   The username of the Keycloak admin that has admin rights to the
-    "entando" realm, e.g entando-keycloak-admin
+Retrieve the following information from the existing Keycloak instance:
 
+-   The username of the Keycloak admin with rights to the "entando" realm, e.g entando-keycloak-admin
 -   The Keycloak admin password, e.g. password123
+-   The base URL for the Keycloak server, including the auth value, e.g. https://YOUR-KEYCLOAK-INSTANCE.com/auth
 
--   The base url for the Keycloak server, including the auth value, e.g.
-    https://your-keycloak-instance.com/auth
+### 2. Generate the Secret
 
-### 2. Generate the secret
-
-You now need to generate a secret with name *keycloak-admin-secret*
-using the information retrieved from step 1. The Entando administrator
-will automatically detect this secret by name, and use it to log onto
-the provided Keycloak server.
-
-Here is an example of the secret you will need to construct:
+Generate a Secret named *keycloak-admin-secret* with the information retrieved in Step 1. For example:
 
     ---
     apiVersion: v1
     stringData:
         username: #the username of the Keycloak admin user for the "entando" realm
-        password: #the password of this Keycloak admin user
-        url: #the base url of the Keycloak service, typically ending with the path /auth
+        password: #the password of the Keycloak admin user
+        url: #the base URL of the Keycloak service, typically ending with the path /auth
     kind: Secret
     metadata
         name: keycloak-admin-secret
         namespace: YOUR-APP-NAMESPACE
     type: Opaque
 
+The client for the admin user will automatically detect this Secret by name and use it to log in to the Keycloak server at the specified URL.
```

Comment:
"The client for the admin user.." -> "Entando will use these admin credentials via the named Secret in order to make use of the Keycloak APIs to provision the entando realm"